toolchain/Toolchain-qurt.cmake

```diff
@@ -57,40 +57,40 @@ set(CROSSDEV "hexagon-")
 # Detect compiler version
 if(${HEXAGON_TOOLS_ROOT} MATCHES "HEXAGON_Tools/6.4.")
 
-# Use the HexagonTools compiler (6.4.06) - This is deprecated
-set(HEXAGON_BIN	${HEXAGON_TOOLS_ROOT}/qc/bin)
-set(HEXAGON_GNU_BIN ${HEXAGON_TOOLS_ROOT}/gnu/bin)
-set(HEXAGON_ISS_DIR ${HEXAGON_TOOLS_ROOT}/lib/iss)
+	# Use the HexagonTools compiler (6.4.06) - This is deprecated
+	set(HEXAGON_BIN	${HEXAGON_TOOLS_ROOT}/qc/bin)
+	set(HEXAGON_GNU_BIN ${HEXAGON_TOOLS_ROOT}/gnu/bin)
+	set(HEXAGON_ISS_DIR ${HEXAGON_TOOLS_ROOT}/lib/iss)
 set(TOOLSLIB ${HEXAGON_TOOLS_ROOT}/dinkumware/lib/${V_ARCH}/G0/pic)
 
-set(CMAKE_C_COMPILER	${HEXAGON_BIN}/${CROSSDEV}clang)
-set(CMAKE_CXX_COMPILER  ${HEXAGON_BIN}/${CROSSDEV}clang++)
+	set(CMAKE_C_COMPILER	${HEXAGON_BIN}/${CROSSDEV}clang)
+	set(CMAKE_CXX_COMPILER  ${HEXAGON_BIN}/${CROSSDEV}clang++)
 
-set(CMAKE_AR	  ${HEXAGON_GNU_BIN}/${CROSSDEV}ar CACHE FILEPATH "Archiver")
-set(CMAKE_RANLIB  ${HEXAGON_GNU_BIN}/${CROSSDEV}ranlib)
-set(CMAKE_NM	  ${HEXAGON_GNU_BIN}/${CROSSDEV}nm)
-set(CMAKE_OBJDUMP ${HEXAGON_GNU_BIN}/${CROSSDEV}objdump)
-set(CMAKE_OBJCOPY ${HEXAGON_GNU_BIN}/${CROSSDEV}objcopy)
-set(HEXAGON_LINK  ${HEXAGON_GNU_BIN}/${CROSSDEV}ld)
+	set(CMAKE_AR	  ${HEXAGON_GNU_BIN}/${CROSSDEV}ar CACHE FILEPATH "Archiver")
+	set(CMAKE_RANLIB  ${HEXAGON_GNU_BIN}/${CROSSDEV}ranlib)
+	set(CMAKE_NM	  ${HEXAGON_GNU_BIN}/${CROSSDEV}nm)
+	set(CMAKE_OBJDUMP ${HEXAGON_GNU_BIN}/${CROSSDEV}objdump)
+	set(CMAKE_OBJCOPY ${HEXAGON_GNU_BIN}/${CROSSDEV}objcopy)
+	set(HEXAGON_LINK  ${HEXAGON_GNU_BIN}/${CROSSDEV}ld)
 set(HEXAGON_ARCH_FLAGS  -march=hexagonv5)
 
 elseif(${HEXAGON_TOOLS_ROOT} MATCHES "HEXAGON_Tools/7.2.")
 
-# Use the HexagonTools compiler (7.2.12) from Hexagon 3.0 SDK
-set(HEXAGON_BIN	${HEXAGON_TOOLS_ROOT}/bin)
-set(HEXAGON_ISS_DIR ${HEXAGON_TOOLS_ROOT}/lib/iss)
+	# Use the HexagonTools compiler (7.2.12) from Hexagon 3.0 SDK
+	set(HEXAGON_BIN	${HEXAGON_TOOLS_ROOT}/bin)
+	set(HEXAGON_ISS_DIR ${HEXAGON_TOOLS_ROOT}/lib/iss)
 set(TOOLSLIB ${HEXAGON_TOOLS_ROOT}/target/hexagon/lib/${V_ARCH}/G0/pic)
 
-set(CMAKE_C_COMPILER	${HEXAGON_BIN}/${CROSSDEV}clang)
-set(CMAKE_CXX_COMPILER  ${HEXAGON_BIN}/${CROSSDEV}clang++)
+	set(CMAKE_C_COMPILER	${HEXAGON_BIN}/${CROSSDEV}clang)
+	set(CMAKE_CXX_COMPILER  ${HEXAGON_BIN}/${CROSSDEV}clang++)
 
-set(CMAKE_AR	  ${HEXAGON_BIN}/${CROSSDEV}ar CACHE FILEPATH "Archiver")
-set(CMAKE_RANLIB  ${HEXAGON_BIN}/${CROSSDEV}ranlib)
-set(CMAKE_NM	  ${HEXAGON_BIN}/${CROSSDEV}nm)
-set(CMAKE_OBJDUMP ${HEXAGON_BIN}/${CROSSDEV}objdump)
-set(CMAKE_OBJCOPY ${HEXAGON_BIN}/${CROSSDEV}objcopy)
-set(HEXAGON_LINK  ${HEXAGON_BIN}/${CROSSDEV}link)
-set(HEXAGON_ARCH_FLAGS  
+	set(CMAKE_AR	  ${HEXAGON_BIN}/${CROSSDEV}ar CACHE FILEPATH "Archiver")
+	set(CMAKE_RANLIB  ${HEXAGON_BIN}/${CROSSDEV}ranlib)
+	set(CMAKE_NM	  ${HEXAGON_BIN}/${CROSSDEV}nm)
+	set(CMAKE_OBJDUMP ${HEXAGON_BIN}/${CROSSDEV}objdump)
+	set(CMAKE_OBJCOPY ${HEXAGON_BIN}/${CROSSDEV}objcopy)
+	set(HEXAGON_LINK  ${HEXAGON_BIN}/${CROSSDEV}link)
+set(HEXAGON_ARCH_FLAGS
```

Comment:
The code at line 104 below is now duplicated in qurt_flags.cmake. It needs to be removed here.